doc/install.rst

```diff
@@ -0,0 +1,373 @@
+Use the default network transfer method. The IVSHMEM method induces
+additional latency that is built into its implementation. When using
+VirtIO for a network device the VM is already using a highly optimized
+memory copy anyway so there is no need to make another one.
+
+If you insist on using IVSHMEM for Scream—despite its inferiority to the
+default network implementation—the Windows Host Application can be told
+what device to use. Create a ``looking-glass-host.ini`` file in the same
+directory as the looking-glass-host.exe file. In it, you can use the
+``os:shmDevice`` option like so:
+
+.. code:: INI
+
+   [os]
+   shmDevice=1
+
+.. _using_the_windows_host_application:
+
+Using the Windows Host Application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Start by downloading the correct version for your release from
+https://looking-glass.io/downloads. You can either choose between
+**Official Releases**, which are stable; or **Release Candidates**, new versions
+about to be stable, but haven't passed validation.
+
+.. note::
+   If your **looking-glass-client** was created by building from the **master
+   branch** you have to pick the **Bleeding Edge** version.
+
+Next, extract the zip archive using the commit hash for the password.
+Then, run the ``looking-glass-host-setup.exe`` installer and install the host.
+By default, the installer will install a service that
+automatically starts the host application at boot. The installer can
+also be installed in silent mode with the ``/S`` switch. You can find other
+command line options with the ``/h`` switch.
+
+The windows host application captures the windows desktop and stuffs the
+frames into the shared memory via the shared memory virtual device,
+without this Looking Glass will not function. It is critical that the
+version of the host application matches the version of the client
+application, as differing versions can be, and usually are,
+incompatible.
+
+.. note::
+   As of 2020-10-23, Microsoft Defender is known to mark the
+   Looking-Glass host executable as a virus and in some cases will
+   automatically delete the file.
+
+.. _running_the_client:
+
+Running the Client
+------------------
+
+The client command is the binary file: **looking-glass-client**. This
+command should run after the Windows Host Application has started.
+
+.. _client_cli_options:
+
+Command Line Options
+~~~~~~~~~~~~~~~~~~~~
+
+A full list of command line options is available with the ``--help`` or ``-h``
+options.
+
+Example: ``looking-glass-client --help``
+
+Common options include ``-s`` for disabling spice, ``-S`` for disabling the
+screen saver, and ``-F`` to automatically enter full screen.
+
+Options may be provided with a Short form, if available, or directly by
+Long form. Boolean options may be specified without a paramater to toggle their
+state.
+
+Examples:
+
+- ``looking-glass-client -F yes`` (Short)
+- ``looking-glass-client win:fullScreen yes`` (Long)
```

Comment:
```suggestion
- ``looking-glass-client win:fullScreen=yes`` (Long)
```

Reply:
incorrect

Reply:
Try it, says `251465269781 [W]             option.c:294  | option_parse                   | Ignored invalid argument: yes`.

Reply:
my bad